assets/terraform-modules/bare-metal/flatcar-linux/kubernetes/cl/install.yaml.tmpl

```diff
@@ -40,9 +39,13 @@ storage:
             -C ${os_channel} \
             -V ${os_version} \
             -o "${container_linux_oem}" \
-            ${baseurl_flag} \
```

Comment:
Hm, I agree about preserving kernel arguments, but I think it's OK that Ignition is only applied during installation. We do not support running Ignition multiple times and I have doubts this is something we want to support. Or at least we should have a conversation about this.